backend/src/openarchiefbeheer/destruction/migrations/0017_destructionlistitem_zaak.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.15 on 2024-08-09 13:48
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def relate_zaak(apps, schema_editor):
+    Zaak = apps.get_model("zaken", "Zaak")
+    DestructionListItem = apps.get_model("destruction", "DestructionListItem")
+
+    items = DestructionListItem.objects.filter(zaak__isnull=True).iterator(
+        chunk_size=1000
+    )
+    updated_items = []
+    for item in items:
+        zaak = Zaak.objects.filter(url=item.zaak_url).first()
+        if not zaak:
+            continue
+
+        item.zaak = zaak
+        updated_items.append(item)
+
+    DestructionListItem.objects.bulk_update(updated_items, fields=["zaak"])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zaken", "0004_alter_zaak_archiefnominatie"),
+        ("destruction", "0016_alter_destructionlistitem_unique_together_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="destructionlistitem",
+            name="zaak",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="items",
+                to="zaken.zaak",
+                verbose_name="zaak",
+            ),
+        ),
+        migrations.RunPython(relate_zaak, migrations.RunPython.noop),
```

Comment:
I think we also need a backwards migration setting `zaak_url` base on `zaak.url`?